### PR TITLE
added french translation

### PR DESCRIPTION
**Pull request by superzazu**
French translation for the project; thanks @Solaarmoon & @metacosm for your work. Feel free to check if everything is okay :)

### CODE REVIEW

**Comment by metacosm** [comment]
Looks good to me. 👍

**Comment by lhc70000** [comment]
Does this contains all features introduced in 0.0.4?

Also, if you don't mind, I prefer waiting for a few days and adding translations needed for 0.0.5 before merge.

**Comment by metacosm** [comment]
The features introduced in 0.0.4 should be translated, yes. I will take a look at what's needed for 0.0.5.

**Comment by lhc70000** [comment]
Please update to latest `develop` branch and read [https://github.com/lhc70000/iina/issues/291#issuecomment-279206928](https://github.com/lhc70000/iina/issues/291#issuecomment-279206928).

**Comment by metacosm** [comment]
Working on it.

File: iina/fr.lproj/KeyBinding.strings [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  iina
+
+  Created by Nicolas Allemand on 12/02/2017.
+  Copyright © 2017 lhc. All rights reserved.
+*/
+
+"cmd.ignore" = "Ignorer";
+"cmd.seek" = "Aller";
+"cmd.set" = "Configurer";
+"cmd.cycle" = "Cycler";
```
Comment (metacosm) [inline iina/fr.lproj/KeyBinding.strings]: Alterner, peut-être?

File: iina/fr.lproj/Localizable.strings [hunk 2]
```diff
@@ -81,14 +81,30 @@
...
+"pl_menu.play_next" = "Écouter ensuite";
+"pl_menu.remove" = "Enlever";
+"pl_menu.remove_multi" = "Enlever la sélection";
+"pl_menu.delete" = "Placer dans la corbeille";
+"pl_menu.delete_multi" = "Placer la sélection dans la corbeille";
+"pl_menu.delete_after_play" = "Placer dans la corbeille après lecture";
+"pl_menu.delete_after_play_multi" = "Placer la sélection dans la corbeille après lecture";
+"pl_menu.reveal_in_finder" = "Afficher dans le Finder";
+"pl_menu.add_item" = "Ajouter...";
+"pl_menu.clear_playlist" = "Vider la playlist";
```
Comment (metacosm) [inline iina/fr.lproj/Localizable.strings]: playlist -> liste de lecture?

File: iina/fr.lproj/Localizable.strings [hunk 3]
```diff
@@ -81,14 +81,30 @@
...
 
 "track.none" = "<Aucun>";
 
-"alert.extra_option_error" = "Erreur lors du setup de l'option --%@=%@ avec une valeur de retour %d. Veuillez vérifier vos options additionnelles dans Préférences > Avancé.";"
+"alert.extra_option_error" = "Erreur lors du setup de l'option --%@=%@ avec une valeur de retour %d. Veuillez vérifier vos options additionnelles dans Préférences > Avancé.";
+"alert.keybinding_config_error" = "Erreur lors de la lecture du fichier de configuration des touches \"%@\" ! Les valeurs par défaut de IINA seront utilisées. Veuillez vérifier votre configuration.";
+
+
+"pl_menu.title_multi" = "%d éléments";
+"pl_menu.play_next" = "Écouter ensuite";
```
Comment (metacosm) [inline iina/fr.lproj/Localizable.strings]: Écouter -> Jouer?

**Review by metacosm — CHANGES_REQUESTED** [review]

File: iina/fr.lproj/KeyBinding.strings (hunk 1 above)
Comment (superzazu) [inline iina/fr.lproj/KeyBinding.strings]: Bonne idée, je n'y avais pas pensé ! :-)

File: iina/fr.lproj/Localizable.strings (hunk 3 above)
Comment (superzazu) [inline iina/fr.lproj/Localizable.strings]: Exact, erreur de ma part.

File: iina/fr.lproj/Localizable.strings (hunk 2 above)
Comment (superzazu) [inline iina/fr.lproj/Localizable.strings]: Au début de la traduction, j'ai effectivement hésité entre ces deux termes; avant de constater qu'Apple utilise le terme "playlist" sur iTunes. Du coup, j'ai gardé le terme pour toute la traduction :)

Comment (metacosm) [inline iina/fr.lproj/Localizable.strings]: 👍

**Comment by superzazu** [comment]
Okay it seems everything is mergeable :-)
Thanks @metacosm !

**Pull request merged**